specification/ai/Azure.AI.Projects/common/models.tsp

```diff
@@ -0,0 +1,140 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+
+namespace Azure.AI.Projects;
+
+@doc("The available sorting options when requesting a list of response objects.")
+union ListSortOrder {
+  string,
+
+  @doc("Specifies an ascending sort order.")
+  ascending: "asc",
+
+  @doc("Specifies a descending sort order.")
+  descending: "desc",
+}
+
+@doc("List View Type Definition")
+union ListViewType {
+  string,
+
+  @doc("List only active items.")
+  ActiveOnly: "ActiveOnly",
+
+  @doc("List only archived items.")
+  ArchivedOnly: "ArchivedOnly",
+
+  @doc("List all items.")
+  All: "All",
+}
+
+alias AssetBase = {
+  @doc("Asset stage")
+  @visibility(Lifecycle.Read, Lifecycle.Create, Lifecycle.Update)
+  stage?: string;
```

Comment:
Is this truly a free-form string value that is also optional? What does it represent? The documentation is not very helpful.